test/unit/memcached_test.rb

```diff
@@ -6,6 +6,13 @@ def to_str
   end
 end
 
+# assert_equal calls obj.respond_to?, which isn't supported by NOT_FOUND < BasicObject
+class Memcached
+  def NOT_FOUND.respond_to?(*)
```

Comment:
This makes me a bit nervous, because we could end up calling it in the library code and not realize it's broken.

Reply:
I took the liberty to push a commit that moves this gem to `minitest` removing the need for that hack.